topcoffea/json/signal_samples/private_UL/UL16APV_tHq_b1.json

```diff
@@ -59,7 +59,7 @@
     "/store/user/kmohrman/FullProduction/FullR2/UL16APV/Round1/Batch1/naodOnly_step/v1/nAOD_step_tHq4f_all22WCsStartPtCheckdim6TopMay20GST_run0/NAOD-00000_8595.root"
   ],
   "nEvents": 7500000,
-  "nGenEvents": 15024000.0,
```

Comment:
@Andrew42, why do we have to get rid of the point 0?

Reply:
that's just how the python `json` module is loading and saving the file. I made no requirements on the typing of the dictionary values when I loaded the json. So this is all a result of how the python module is loading and saving the file.

Reply:
Ok, I don't think we use this number for anything anyway, so probably it's not important. I'm always just a big paranoid about integers because if they get included in a calculation at some point, python might accidental think we want the result to be an integer....